tests/fixtures/celery.py

```diff
@@ -77,6 +93,20 @@ class CeleryFixture:
     worker: WorkController
     session_maker: MockSessionMaker
 
+    @contextmanager
+    def patch_retry_backoff(self) -> Generator[CeleryRetriesMock, None, None]:
```

Comment:
I needed a way to test a celery task that used `autoretry_for` and `retry_backoff` without actually waiting for the backoffs to occur in the task. It seems like this is useful behaviour generally that we will want again, so I added it to the celery fixture, instead of just doing it in the test.